### PR TITLE
In error, the first letter is low-case letter

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: YuPengZTE yu.peng36@zte.com.cn

### CODE REVIEW

File: libcontainerd/client_windows.go [hunk 1]
```diff
@@ -359,12 +359,12 @@ func (clnt *client) Resize(containerID, processFriendlyName string, width, heigh
 
 // Pause handles pause requests for containers
 func (clnt *client) Pause(containerID string) error {
-	return errors.New("Windows: Containers cannot be paused")
+	return errors.New("windows: Containers cannot be paused")
```
Comment (thaJeztah) [inline libcontainerd/client_windows.go]: This should remain uppercase, as the product name is `Windows`

File: libcontainerd/client_windows.go [hunk 2]
```diff
@@ -359,12 +359,12 @@ func (clnt *client) Resize(containerID, processFriendlyName string, width, heigh
...
 }
 
 // Resume handles resume requests for containers
 func (clnt *client) Resume(containerID string) error {
-	return errors.New("Windows: Containers cannot be paused")
+	return errors.New("windows: Containers cannot be paused")
```
Comment (thaJeztah) [inline libcontainerd/client_windows.go]: Same here; this should remain uppercase, as the product name is `Windows`

**Review by thaJeztah — CHANGES_REQUESTED** [review]
thanks @YuPengZTE, I left two comments

File: libcontainerd/client_windows.go (hunk 1 above)
Comment (YuPengZTE) [inline libcontainerd/client_windows.go]: Thanks @thaJeztah .
It has been restored.
PTAL.

File: libcontainerd/client_windows.go (hunk 2 above)
Comment (YuPengZTE) [inline libcontainerd/client_windows.go]: Thanks @thaJeztah .
It has been restored.
PTAL.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**